ledger-service/jwt/src/main/scala/com/digitalasset/jwt/Main.scala

```diff
@@ -32,9 +32,8 @@ object Main {
             e.printStackTrace()
             sys.exit(ErrorCodes.GenerateKeysError)
         }
-      case Some(Config(None, Some(GenerateJwt(Some(publicKey), Some(privateKey))))) =>
-        JwtGenerator.generate(
-          domain.KeyPair(publicKey = Seq.empty[Byte], privateKey = Seq.empty[Byte])) match {
+      case Some(Config(None, Some(GenerateJwt(Some(publicKey @ _), Some(privateKey @ _))))) =>
+        JwtGenerator.generate match {
```

Comment:
@leo-da We're just a little more explicit that the contents of the argument are ignored, now. How does this sound to you?

Reply:
I should have removed this `main`... It was supposed to be the tool to generate keys for the JWT, but I don't think we are using it right now.

In any case regarding the `case Some(Config(None, Some(GenerateJwt(Some(publicKey @ _), Some(privateKey @ _)))))`
you can still used `publicKey` and `privateKey` values, right? Nothing prevents you from this. Why not just use `_` instead. But it reads better like `publicKey @ _`, kind of self-documenting.

Reply:
When you don't use a pattern variable, `-Ywarn-unused` suggests this form, for exactly the reason you say.